packages/app/src/components/Survey/SurveyPresentational.tsx

```diff
@@ -99,15 +100,49 @@ export const SurveyPresentational: React.FC<SurveyPresentationalProps> = ({
 
                 <Box>
                     <Heading use="h3" marginBottom="major-3">
-                        {intl.formatMessage({
-                            id: "statistics",
-                            defaultMessage: "Statistics"
-                        })}
+                        Statistics
```

Comment:
Why is the `intl` message removed? was it intentional?

Reply:
Also for the texts below, do we expect to use intl or simply support english lang only?